custom/src/main/java/co/elastic/otel/dynamicconfig/DynamicConfiguration.java

```diff
@@ -84,6 +84,54 @@ private void initSendingStates() {
     }
   }
 
+  /** Can be executed repeatedly even if sending is currently stopped */
+  public void stopSendingSpans() {
```

Comment:
Is there a particular reason to having start/stop methods instead of just a `setSendingSpans(boolean)` method?